boilerplate/app/components/TextField.tsx

```diff
@@ -246,21 +244,31 @@ const $inputStyle: TextStyle = {
   fontFamily: typography.primary.normal,
   color: colors.text,
   fontSize: 16,
-  height: 24, // to make the height consistent on Android and iOS
+  height: 24,
   // https://github.com/facebook/react-native/issues/21720#issuecomment-532642093
   paddingTop: 0,
   paddingBottom: 0,
   paddingLeft: 0,
   paddingRight: 0,
-  marginTop: 0,
-  marginBottom: 0,
-  marginLeft: 0,
-  marginRight: 0,
+  marginTop: 8,
```

Comment:
Any reason not to use the spacing scale throughout?

Reply:
I will be updating this throughout the app starting with demo screens https://github.com/infinitered/ignite/issues/2068 @robinheinze

Reply:
@kateinkim please take a look at #2067 since I handle some RTL styling there. that PR gets merged here, so when these two are merged you should be good to go

Reply:
Yep I will be waiting on it until it's merged! Thanks @mazenchami

Reply:
> Any reason not to use the spacing scale throughout?

@robinheinze no reason, just creating more work for y'all! :) tbh though, i just haven't done it since we had the array thing. @kateinkim do you want me to update this PR with the spacing or you'll be going over the entire repo changing this?

Reply:
I can go over the entire repo :)